frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,119 @@
-import { Box } from '@mui/material';
+import {
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Button,
+  FormControl,
+} from '@mui/material';
+import Background from '../components/Background';
+import { useState } from 'react';
 
 export const UpdateUser = () => {
-  return <Box>This is the UpdateUser Page!</Box>;
+  const [userDetails, setUserDetails] = useState<{
+    emojis: string;
+    faculty: string;
+    bestEat: string;
+    interests: string;
+  }>({ emojis: '', faculty: '', bestEat: '', interests: '' });
+
+  const handleChange = (e: any) => {
+    const { name, value } = e.target;
+    setUserDetails({
+      ...userDetails,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    console.log(userDetails);
+    // TODO:
+    // Hit the correct API to successfully store info to the db
+    const res = await fetch('http://dynamodb:8000/updateUser', {
+      method: 'POST',
+    });
+    const data = await res.json();
+    console.log(data);
+  };
+
+  const baseFlexStyles = {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  };
+
+  return (
+    <Background bgcolor={'#fff'}>
+      <Grid item xs={12} sx={baseFlexStyles}>
+        <Typography variant={'h5'}>This is...</Typography>
+        <Typography variant={'h2'}>TuesHey</Typography>
+        <Typography variant={'h4'} mb={7}>
+          Let's get to know you ✌️😙✌️
+        </Typography>
+
+        <div style={{ margin: '1em 0' }}>
+          <InputLabel sx={{ marginBottom: '8px' }}>
+            Describe yourself in 3 emojis!
+          </InputLabel>
+          <TextField
```

Comment:
Each form item is a different size, we want to make sure they are the identical width. To do this, we need to make sure that all our components are either wrapped with FormControl OR are wrapped with FormControl under the hood (like TextField). Here's a hint on what your form Grid could look like:

```
<Grid item container xs={12} paddingX={5} style={{ gap: 15 }}>
        <TextField
          fullWidth
          id="outlined-basic"
          label="Describe yourself in 3 emojis!"
          variant="outlined"
          placeholder={'😎 ⚽ 🍔'}
        />

        <FormControl fullWidth>
          <InputLabel id="demo-simple-select-label">Age</InputLabel>
          <Select
            labelId="demo-simple-select-label"
            id="demo-simple-select"
            value={userDetails.faculty}
            label="Age"
            onChange={handleChange}
          >
            <MenuItem value={10}>Ten</MenuItem>
            <MenuItem value={20}>Twenty</MenuItem>
            <MenuItem value={30}>Thirty</MenuItem>
          </Select>
        </FormControl>
      </Grid>
```